### PR TITLE
feat: Sync multisig output cell.

### PR DESCRIPTION
**Pull request by yanguoyu**

### CODE REVIEW

**Comment by Keith-CY** [comment]
Balances are not cleared by `clear cache`

File: packages/neuron-wallet/src/utils/rpc-request.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    params?: any
+  }[]
+): Promise<any[]> => {
+  const res = await axios({
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json'
+    },
+    data: options.map(v => ({
+      id: 0,
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/utils/rpc-request.ts:16]: `id` should be different in the array, it is used to correlate request and response among the request array and response array.

**Comment by yanguoyu** [comment]
> Balances are not cleared by `clear cache`

Fixed it by https://github.com/nervosnetwork/neuron/pull/2332/commits/07038ea658c87a59b0314f3d2df7ffc15da418fa

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-wallet/src/services/multisig.ts [hunk 2]
```diff
@@ -62,16 +71,219 @@ export default class MultisigService {
...
+              addressCursorMap.get(v.fullPayload)
+            ]
+          }
+        })
+      )
+      const nextMultisigConfigs: MultisigConfig[] = []
+      res.forEach((v, idx) => {
+        if (!v.error && v?.result?.objects?.length) {
+          addressCursorMap.set(currentMultisigConfigs[idx].fullPayload, v?.result?.last_cursor)
+          liveCells.push(...v.result.objects.map((output: any) => MultisigOutput.fromIndexer(output)))
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/multisig.ts:124]: cells with `type script` should be eliminated

Comment (yanguoyu) [reply packages/neuron-wallet/src/services/multisig.ts:124]: https://github.com/nervosnetwork/neuron/pull/2340

**Pull request merged**